scenario/deploy.xml

```diff
@@ -69,6 +70,7 @@
                 <item name="lock" xsi:type="object" priority="1000">Magento\MagentoCloud\Step\Deploy\InstallUpdate\ConfigUpdate\Lock</item>
                 <!-- Config update end -->
                 <item name="config-import" xsi:type="object" priority="1100">Magento\MagentoCloud\Step\Deploy\InstallUpdate\Install\ConfigImport</item>
+                <item name="split-db" xsi:type="object" priority="1100">Magento\MagentoCloud\Step\Deploy\SplitDbConnection</item>
```

Comment:
The same priority is in the previous step, please use a different value. For example `1200` and update to `1300` for the next step.

Reply:
Done